macros/plotCTSEvent.C

```diff
@@ -0,0 +1,154 @@
+#include <TTree.h>
+#include <TFile.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <Rtypes.h>
+#include <TCanvas.h>
```

Comment:
Needed?
Btw: I don't even remember if this is not also an unneeded remnant in the example macro. :wink: